src/main/java/com/julienviet/pgclient/impl/PostgresSQLConnection.java

```diff
@@ -30,81 +38,108 @@ public PostgresSQLConnection(DbConnection conn) {
 
   @Override
   public SQLConnection setAutoCommit(boolean autoCommit, Handler<AsyncResult<Void>> handler) {
-    throw new UnsupportedOperationException();
+    inAutoCommit.set(autoCommit);
```

Comment:
I think that we should rethink this part, because with pipelining it is different than with the other driver